CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to
 
 ## Added
 
+- Add a route to reorder target_courses
```

Comment:
I would like to question the necessity of a specific route for this purpose.
If you update the position field of a target course, the normal way, the backend can deduce that ordering is affected and do what is needed?
Are there sufficient advantages to do it via a specific route to justify the additional complexity?